slides/git-intro/index.html

```diff
@@ -131,6 +139,22 @@
                     <p>We won't be looking at those just yet....</p>
                 </section>
 
+                <section>
+                    <h2>Quick Reference</h2>
+                    <ul class="glossary">
+                        <li><strong>Repository</strong> - A collection of files and references to changes.</li>
+                        <li><strong>Branch</strong> - An isolated working tree.</li>
+                        <li><strong>Commit</strong> - A change to a file or set of files.</li>
+                        <li><strong>Clone</strong> - Copying a repository from remote to local.</li>
+                        <li><strong>Push</strong> - Move local branch to remote counterpart.</li>
+                        <li><strong>Pull</strong> - Copy remote branch into local branch</li>
+                        <li><strong>Merge</strong> - Combine two branches, preserving changes in both.</li>
+                        <li><strong>Conflict</strong> - Conflicting changes in two branches to be merged.</li>
+                        <li><strong>Checkout</strong> - Switch working tree to another branch.</li>
+                        <li><strong>Diff</strong> - The difference between two branches to be merged.</li>
+                    </ul>
```

Comment:
By no means finite, but a reasonable definition list for those with no experience of Git, or indeed, version control...